index.js

```diff
@@ -1,159 +1,98 @@
+        'selector-max-universal': 1,
+        'selector-max-specificity': '0,3,1',
+        'value-keyword-case': [
+            'lower',
             {
-                "severity": "warning"
+                camelCaseSvgKeywords: true,
             }
         ],
-        "selector-max-combinators": [
-            2,
+        'scss/at-function-pattern': [
+            '^[a-z]+((\\d)|([A-Z0-9][a-z0-9]+))*([A-Z])*$',
             {
-                "severity": "warning"
-            }
+                message: 'Expected function name to be camelCase',
+            },
         ],
-        "selector-max-universal": 1,
-        "selector-max-specificity": "0,3,0",
-        "selector-list-comma-newline-before": "never-multi-line",
-        "selector-list-comma-space-after": "always-single-line",
-        "custom-media-pattern": "^[a-z][a-z-A-Z0-9]*$",
-        "media-feature-parentheses-space-inside": "never",
-        "media-query-list-comma-newline-before": "never-multi-line",
-        "at-rule-semicolon-space-before": "never",
-        "indentation": 4,
-        "max-line-length": 120,
-        "no-descending-specificity": null,
-        "no-duplicate-selectors": true,
-        "no-empty-source": true,
-        "block-no-empty": true,
-        "color-hex-length": "long",
-        "color-hex-case": ["upper"],
-        "rule-empty-line-before": [
-            "always",
+        'scss/at-mixin-argumentless-call-parentheses': 'always',
+        'scss/at-mixin-pattern': [
+            '^[a-z]+((\\d)|([A-Z0-9][a-z0-9]+))*([A-Z])*$',
             {
-                "except": [
-                    "first-nested"
-                ],
-                "ignore": [
-                    "after-comment"
-                ]
-            }
+                message: 'Expected mixin to be camelCase',
+            },
         ],
-        "shorthand-property-no-redundant-values": true,
-        "scss/dollar-variable-pattern": "^[a-z][a-z-A-Z0-9]*$",
-        "scss/dollar-variable-colon-space-after": "always-single-line",
-        "scss/dollar-variable-colon-space-before": "never",
-        "scss/dollar-variable-empty-line-before": [
-            "always",
+        'scss/declaration-nested-properties': 'never',
+        'scss/dollar-variable-pattern': [
+            '^[a-z]+((\\d)|([A-Z0-9][a-z0-9]+))*([A-Z])*$',
             {
-                "except": [
-                    "first-nested",
-                    "after-comment",
-                    "after-dollar-variable"
-                ]
-            }
+                message: 'Expected variable to be camelCase',
+            },
+        ],
+        'scss/no-global-function-names': null,
+        'scss/percent-placeholder-pattern': [
+            '^[a-z][a-z-A-Z0-9]*$',
+            {
+                message: 'Expected placeholder to be camelCase',
+            },
         ],
-        "scss/percent-placeholder-pattern": "^[a-z][a-z-A-Z0-9]*$",
-        "scss/double-slash-comment-whitespace-inside": "always",
```

Comment:
why this rules removed are they already tackled by the extends?

Reply:
@alexander-schranz Yes, I've checked all of them again. All the one's I've removed are already defined in the extended set.